### PR TITLE
add basic hyprland support

### PR DESCRIPTION
**Pull request by TotalChaos05**
as of now, this only adds window borders, ill work on group borders later

### CODE REVIEW

**Comment by TotalChaos05** [comment]
the flake.lock also needs to be updated for this PR. as the hyprland module is relatively new

File: modules/hyprland/hm.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+let
+  text = base05;
+  urgent = base08;
+  focused = base0A;
+  unfocused = base03;
+
+  fonts = {
+    names = [ config.stylix.fonts.sansSerif.name ];
+    size = config.stylix.fonts.sizes.desktop + 0.0;
+  };
```
Comment (danth) [inline modules/hyprland/hm.nix:14]: Looks like some of these variables aren't used, could you remove them?

**Review by danth — COMMENTED** [review]
Good to see Home Manager has a built in module for Hyprland now, that makes things easier as we don't have to do anything special to support it.

I'll update `flake.lock` on this branch for the tests to pass.

**Review by danth — APPROVED** [review]
Thanks for your work on this, I finished it off so we can get it merged :)

**Pull request merged**